snippets/al.json

```diff
@@ -262,7 +262,7 @@
     "Snippet: Caption ML": {
         "prefix": "tcaptionml",
         "body": [
-            "CaptionML = ENU = \"\"",
```

Comment:
Could you please add placeholders for the English text and the other text? That way, after adding the snippet the cursor will be placed at the right position and after tabbing will switch to the next placeholder.
Placheolder format is ${placeholderName}